template/lobby.html

```diff
@@ -34,7 +34,9 @@
             view.addEventListener('action', (event) => {
               ipc.send('asynchronous-message', { type: 'action', data: event.detail })
             })
-            document.getElementById('main-view').appendChild(view)
+            let mainview = document.getElementById('main-view')
+            while (mainview.hasChildNodes()) mainview.removeChild(mainview.lastChild);
```

Comment:
Because of the way polymer registers elements, we won't be able to dynamically load more than one game board. If the same game is started again, this codepath will work fine, but if we are switching to a different game board, we need to reload the page in order to purge the currently dynamically loaded template. `location.reload()` is the way to do that.

More specifically, above, we must first remember which game template was previously loaded:

```
let loadedGame
```

Then, after the server sends us the game information:

```
if (game.gamePath == loadedGame) {
  location.href = 'something-to-auto-start-game'
  location.reload()
} else {
  loadedGame = game.gamePath
[...]
```

That's going to be more testable once we have more than one board.